src/editor/panels/BottomPanel.js

```diff
@@ -189,6 +189,7 @@ class BottomPanel {
     $id('opacity').addEventListener('change', this.handleOpacity.bind(this))
     $id('fill_color').init(i18next)
     $id('stroke_color').init(i18next)
+    $id('zoom').shadowRoot.querySelector('elix-number-spin-box').addEventListener('change', (e) => this.changeZoom.bind(this)(e.detail.value))
```

Comment:
It is not optimal to go to the shadow root of the element to capture this message. 
We would need to go to the zoom component to capture the event and transfer it as a "change" event which is already trapped in BottomPanel.js. (line 182)
Can you review this?

Reply:
> Thank you for the PR submission. The CI is failing. Did you run the tests locally (npm run test)?

I requested a "re-run" of the job and it was OK so forget this comment. We still need to fix the previous comment before merging. thanks

Reply:
[d89c79d](https://github.com/SVG-Edit/svgedit/pull/739/commits/d89c79d2521890ab7f52a62e0614d8e30dd1794a)
I revised the way it handles input. Created a proxy listener which handles NumberSpinBox inputs properly and redirects them to zoom component.